tests/find_cmd_tests.rs

```diff
@@ -518,3 +518,50 @@ fn expression_empty_parentheses() {
         ))
         .stdout(predicate::str::is_empty());
 }
+
+#[test]
+#[serial(working_dir)]
+fn find_newer_xy() {
+    #[cfg(target_os = "linux")]
+    let x_options = ["a", "c", "m"];
+    #[cfg(not(target_os = "linux"))]
+    let x_options = ["a", "B", "c", "m"];
+    #[cfg(target_os = "linux")]
+    let y_options = ["a", "c", "m"];
+    #[cfg(not(target_os = "linux"))]
+    let y_options = ["a", "B", "c", "m"];
+
+    for &x in &x_options {
+        for &y in &y_options {
+            let arg = &format!("-newer{x}{y}").to_string();
```

Comment:
Calling `to_string()` is unnecessary because `format!` already returns a string.
```suggestion
            let arg = &format!("-newer{x}{y}");
```